### PR TITLE
Javascript javascript3 week1

### PR DESCRIPTION
**Pull request by shobanamg**

### CODE REVIEW

File: javascript/javascript3/week1/classwork/dogFan/script.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+const getRandomImage = async () => {
+  const resp = await fetch("https://dog.ceo/api/breeds/list/all");
+  const data = await resp.json();
+  const breedArr = Object.keys(data.message);
+  const randomIndex = Math.floor(Math.random() * breedArr.length);
+  const randomBreed = breedArr[randomIndex];
+  const url = `https://dog.ceo/api/breed/${randomBreed}/images/random`;
+  const response = await fetch(url);
+  const breed = await response.json();
+  const breedImgContainer = document.querySelector(".breed-image-container");
```
Comment (naresh699) [inline javascript/javascript3/week1/classwork/dogFan/script.js:26]: try template string to write HTML

File: javascript/javascript3/week1/classwork/mealOrder/script.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+console.log(mealOrder1);
+console.log(mealOrder1[mealOrder1.length - 1].name);
+console.log(mealOrder1[mealOrder1.length - 1].price);
+
+const mealOrderString = JSON.stringify(mealOrder1);
+console.log(mealOrderString);
+console.log(mealOrderString[mealOrderString.length - 1].name);
+
+const mealOrderString2 =
+  '[{"name":"pizza","id":112,"Price":100,"drinks":["Sprite","Fanta"],"extras":"cheese"},{"name":"FriedRice","id":113,"price":120,"drinks":["Sprite","Fanta"],"extras":"lettuce"}]';
```
Comment (naresh699) [inline javascript/javascript3/week1/classwork/mealOrder/script.js:40]: format code in readable format. currently this array is bit complicated to read

**Review by naresh699 — APPROVED** [review]
Please check feedback and merge

File: javascript/javascript3/week1/classwork/dogFan/script.js (hunk 1 above)
Comment (shobanamg) [reply javascript/javascript3/week1/classwork/dogFan/script.js:26]: Hi Naresh
I don't understand what you mean here. can you explain please.

**Pull request merged**